aws-services/reference/package-values.hbs.md

```diff
@@ -28,3 +28,20 @@ The following table lists configuration that applies to the `postgresql` service
 | postgresql.provider_config_ref.name                   | `default`             | string  | Name of your ProviderConfig for the `postgresql` service                                                                                                                                                                                                                               |
 | postgresql.region                                     | `us-east-1`           | string  | The AWS region to create databases in                                                                                                                                                                                                                                                  |
+
+## <a id="mysql"></a> MySQL
+
+The following table lists configuration that applies to the `mysql` service.
+
+| KEY                                              | DEFAULT               | TYPE    | DESCRIPTION                                                                                                                                                                                                                                                                            |
+-------------------------------------------------- | --------------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| mysql.enabled                                    | `false`               | boolean | Enables the MySQL service class                                                                                                                                                                                                                                                   |
+| mysql.infrastructure.security_groups             | _n/a_                 | array   | Security groups for your MySQL databases to belong to                                                                                                                                                                                                                             |
+| mysql.infrastructure.subnet_group.name           | `""`                  | string  | Subnet group for your MySQL databases                                                                                                                                                                                                                                             |
```

Comment:
I think it's worth adding description for `security_groups` and `subnet_group` so we are not leaving them blank.

Reply:
They have one 🤔

Reply:
My bad. Misread the column